PersonDataRequest/PersonDataRequest/Controller/DataViewController.swift

```diff
@@ -11,13 +11,20 @@ class DataViewController: UIViewController {
     
     var selectedName: String?
     
+    var gallery: Gallery = Gallery()
```

Comment:
[Question] Why do you need this Gallery class? Does it give you benefits? Could you just add an array of pictures here?